addons/account/models/chart_template.py

```diff
@@ -538,6 +538,11 @@ class AccountTaxTemplate(models.Model):
         string='Tax Received Account',
         domain=[('deprecated', '=', False)],
         help='Account used as counterpart for the journal entry, for taxes eligible based on payments.')
+    cash_basis_base_account_id = fields.Many2one(
+        'account.account',
+        domain=[('deprecated', '=', False)],
+        string='Base Tax Received Account',
```

Comment:
not sure that string is clear enough... maybe we should have something like
```
Cash basis entry: account for the base
Cash basis entry: account for the tax
```
what daya think, @lap-odoo ?

Reply:
@qdp-odoo @smetl Let me check it on the branch to have a better "feeling". I've launched the rebuild, should give an answer monday